I see error in one test which can be triggered by this change.
 Could you take care of this failing test? 
 AssertionError: Timed out retrying: Expected to find content: 'Add annotation layer' but never did.

--- a/superset-frontend/cypress-base/cypress/integration/chart_list/list_view.test.ts
+++ b/superset-frontend/cypress-base/cypress/integration/chart_list/list_view.test.ts
@@ -30,11 +30,11 @@ describe('chart list view', () => {
     cy.get('[data-test="listview-table"]').should('be.visible');
     // check chart list view header
     cy.get('[data-test="sort-header"]').eq(1).contains('Chart');
-    cy.get('[data-test="sort-header"]').eq(2).contains('Visualization Type');
+    cy.get('[data-test="sort-header"]').eq(2).contains('Visualization type');
     cy.get('[data-test="sort-header"]').eq(3).contains('Dataset');
-    cy.get('[data-test="sort-header"]').eq(4).contains('Modified By');
-    cy.get('[data-test="sort-header"]').eq(5).contains('Last Modified');
-    cy.get('[data-test="sort-header"]').eq(6).contains('Created By');
+    cy.get('[data-test="sort-header"]').eq(4).contains('Modified by');
+    cy.get('[data-test="sort-header"]').eq(5).contains('Last modified');
+    cy.get('[data-test="sort-header"]').eq(6).contains('Created by');
     cy.get('[data-test="sort-header"]').eq(7).contains('Actions');
     cy.get('[data-test="table-row"]').should('have.length', 25);
   });

--- a/superset-frontend/cypress-base/cypress/integration/explore/visualizations/line.test.ts
+++ b/superset-frontend/cypress-base/cypress/integration/explore/visualizations/line.test.ts
@@ -35,10 +35,10 @@ describe('Visualization > Line', () => {
 
   it('should preload mathjs', () => {
     cy.get('script[src*="mathjs"]').should('have.length', 1);
-    cy.contains('Add Annotation Layer').scrollIntoView().click();
+    cy.contains('Add annotation layer').scrollIntoView().click();
     // should not load additional mathjs
     cy.get('script[src*="mathjs"]').should('have.length', 1);
-    cy.contains('Layer Configuration');
+    cy.contains('Layer configuration');
   });
 
   it('should not show validator error when metric added', () => {

--- a/superset-frontend/spec/javascripts/views/CRUD/annotation/AnnotationModal_spec.jsx
+++ b/superset-frontend/spec/javascripts/views/CRUD/annotation/AnnotationModal_spec.jsx
@@ -80,12 +80,12 @@ describe('AnnotationModal', () => {
     const addWrapper = await mountAndWait({});
     expect(
       addWrapper.find('[data-test="annotaion-modal-title"]').text(),
-    ).toEqual('Add Annotation');
+    ).toEqual('Add annotation');
   });
 
   it('renders edit header when annotation prop is included', () => {
     expect(wrapper.find('[data-test="annotaion-modal-title"]').text()).toEqual(
-      'Edit Annotation',
+      'Edit annotation',
     );
   });
 

--- a/superset-frontend/spec/javascripts/views/CRUD/annotationlayers/AnnotationLayerModal_spec.jsx
+++ b/superset-frontend/spec/javascripts/views/CRUD/annotationlayers/AnnotationLayerModal_spec.jsx
@@ -73,13 +73,13 @@ describe('AnnotationLayerModal', () => {
     const addWrapper = await mountAndWait({});
     expect(
       addWrapper.find('[data-test="annotation-layer-modal-title"]').text(),
-    ).toEqual('Add Annotation Layer');
+    ).toEqual('Add annotation layer');
   });
 
   it('renders edit header when layer prop is included', () => {
     expect(
       wrapper.find('[data-test="annotation-layer-modal-title"]').text(),
-    ).toEqual('Edit Annotation Layer Properties');
+    ).toEqual('Edit annotation layer properties');
   });
 
   it('renders input element for name', () => {

--- a/superset-frontend/spec/javascripts/views/CRUD/csstemplates/CssTemplateModal_spec.jsx
+++ b/superset-frontend/spec/javascripts/views/CRUD/csstemplates/CssTemplateModal_spec.jsx
@@ -74,13 +74,13 @@ describe('CssTemplateModal', () => {
     const addWrapper = await mountAndWait({});
     expect(
       addWrapper.find('[data-test="css-template-modal-title"]').text(),
-    ).toEqual('Add CSS Template');
+    ).toEqual('Add CSS template');
   });
 
   it('renders edit header when css template prop is included', () => {
     expect(
       wrapper.find('[data-test="css-template-modal-title"]').text(),
-    ).toEqual('Edit CSS Template Properties');
+    ).toEqual('Edit CSS template properties');
   });
 
   it('renders input elements for template name', () => {

--- a/superset-frontend/src/SqlLab/components/TemplateParamsEditor.jsx
+++ b/superset-frontend/src/SqlLab/components/TemplateParamsEditor.jsx
@@ -122,7 +122,7 @@ export default class TemplateParamsEditor extends React.Component {
       : 0;
     return (
       <ModalTrigger
-        modalTitle={t('Template Parameters')}
+        modalTitle={t('Template parameters')}
         triggerNode={
           <div tooltip={t('Edit template parameters')} buttonSize="small">
             {`${t('Parameters')} `}

--- a/superset-frontend/src/SqlLab/reducers/getInitialState.js
+++ b/superset-frontend/src/SqlLab/reducers/getInitialState.js
@@ -41,7 +41,7 @@ export default function getInitialState({
   const defaultQueryEditor = {
     id: null,
     loaded: true,
-    title: t('Untitled Query'),
+    title: t('Untitled query'),
     sql: 'SELECT *\nFROM\nWHERE',
     selectedText: null,
     latestQueryId: null,

--- a/superset-frontend/src/explore/components/controls/AnnotationLayer.jsx
+++ b/superset-frontend/src/explore/components/controls/AnnotationLayer.jsx
@@ -698,7 +698,7 @@ export default class AnnotationLayer extends React.PureComponent {
             <PopoverSection
               isSelected
               onSelect={() => {}}
-              title={t('Layer Configuration')}
+              title={t('Layer configuration')}
               info={t('Configure the basics of your Annotation Layer.')}
             >
               <TextControl

--- a/superset-frontend/src/explore/components/controls/AnnotationLayerControl.jsx
+++ b/superset-frontend/src/explore/components/controls/AnnotationLayerControl.jsx
@@ -188,7 +188,7 @@ class AnnotationLayerControl extends React.PureComponent {
             trigger="click"
             placement="right"
             content={this.renderPopover(addLayerPopoverKey, addedAnnotation)}
-            title={t('Add Annotation Layer')}
+            title={t('Add annotation layer')}
             visible={this.state.popoverVisible[addLayerPopoverKey]}
             destroyTooltipOnHide
             onVisibleChange={visible =>
@@ -200,7 +200,7 @@ class AnnotationLayerControl extends React.PureComponent {
                 data-test="add-annotation-layer-button"
                 className="fa fa-plus"
               />{' '}
-              &nbsp; {t('Add Annotation Layer')}
+              &nbsp; {t('Add annotation layer')}
             </ListGroupItem>
           </Popover>
         </ListGroup>

--- a/superset-frontend/src/views/CRUD/alert/AlertList.tsx
+++ b/superset-frontend/src/views/CRUD/alert/AlertList.tsx
@@ -210,7 +210,7 @@ function AlertList({
             ? moment.utc(lastEvalDttm).local().format(DATETIME_WITH_TIME_ZONE)
             : '',
         accessor: 'last_eval_dttm',
-        Header: t('Last Run'),
+        Header: t('Last run'),
         size: 'lg',
       },
       {
@@ -242,7 +242,7 @@ function AlertList({
             <RecipientIcon key={r.id} type={r.type} />
           )),
         accessor: 'recipients',
-        Header: t('Notification Method'),
+        Header: t('Notification method'),
         disableSortBy: true,
         size: 'xl',
       },
@@ -289,7 +289,7 @@ function AlertList({
             canEdit
               ? {
                   label: 'execution-log-action',
-                  tooltip: t('Execution Log'),
+                  tooltip: t('Execution log'),
                   placement: 'bottom',
                   icon: 'note' as IconName,
                   onClick: handleGotoExecutionLog,
@@ -344,7 +344,7 @@ function AlertList({
   }
   if (canDelete) {
     subMenuButtons.push({
-      name: t('Bulk Select'),
+      name: t('Bulk select'),
       onClick: toggleBulkSelect,
       buttonStyle: 'secondary',
       'data-test': 'bulk-select-toggle',
@@ -365,7 +365,7 @@ function AlertList({
   const filters: Filters = useMemo(
     () => [
       {
-        Header: t('Created By'),
+        Header: t('Created by'),
         id: 'created_by',
         input: 'select',
         operator: FilterOperators.relationOneMany,
@@ -408,7 +408,7 @@ function AlertList({
     <>
       <SubMenu
         activeChild={pathName}
-        name={t('Alerts & Reports')}
+        name={t('Alerts & reports')}
         tabs={[
           {
             name: 'Alerts',

--- a/superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx
+++ b/superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx
@@ -69,15 +69,15 @@ const CONDITIONS = [
     value: '>=',
   },
   {
-    label: t('== (Is Equal)'),
+    label: t('== (Is equal)'),
     value: '==',
   },
   {
-    label: t('!= (Is Not Equal)'),
+    label: t('!= (Is not equal)'),
     value: '!=',
   },
   {
-    label: t('Not Null'),
+    label: t('Not null'),
     value: 'not null',
   },
 ];
@@ -1060,7 +1060,7 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
         <div className="header-section">
           <StyledInputContainer>
             <div className="control-label">
-              {isReport ? t('Report Name') : t('Alert Name')}
+              {isReport ? t('Report name') : t('Alert name')}
               <span className="required">*</span>
             </div>
             <div className="input-container">
@@ -1068,7 +1068,7 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
                 type="text"
                 name="name"
                 value={currentAlert ? currentAlert.name : ''}
-                placeholder={isReport ? t('Report Name') : t('Alert Name')}
+                placeholder={isReport ? t('Report name') : t('Alert name')}
                 onChange={onTextChange}
               />
             </div>
@@ -1114,7 +1114,7 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
           {!isReport && (
             <div className="column condition">
               <StyledSectionTitle>
-                <h4>{t('Alert Condition')}</h4>
+                <h4>{t('Alert condition')}</h4>
               </StyledSectionTitle>
               <StyledInputContainer>
                 <div className="control-label">
@@ -1210,8 +1210,8 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
             <StyledSectionTitle>
               <h4>
                 {isReport
-                  ? t('Report Schedule')
-                  : t('Alert Condition Schedule')}
+                  ? t('Report schedule')
+                  : t('Alert condition schedule')}
               </h4>
             </StyledSectionTitle>
             <AlertReportCronScheduler
@@ -1221,11 +1221,11 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
               onChange={newVal => updateAlertState('crontab', newVal)}
             />
             <StyledSectionTitle>
-              <h4>{t('Schedule Settings')}</h4>
+              <h4>{t('Schedule settings')}</h4>
             </StyledSectionTitle>
             <StyledInputContainer>
               <div className="control-label">
-                {t('Log Retention')}
+                {t('Log retention')}
                 <span className="required">*</span>
               </div>
               <div className="input-container">
@@ -1249,7 +1249,7 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
             </StyledInputContainer>
             <StyledInputContainer>
               <div className="control-label">
-                {t('Working Timeout')}
+                {t('Working timeout')}
                 <span className="required">*</span>
               </div>
               <div className="input-container">
@@ -1264,7 +1264,7 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
               </div>
             </StyledInputContainer>
             <StyledInputContainer>
-              <div className="control-label">{t('Grace Period')}</div>
+              <div className="control-label">{t('Grace period')}</div>
               <div className="input-container">
                 <input
                   type="number"
@@ -1279,7 +1279,7 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
           </div>
           <div className="column message">
             <StyledSectionTitle>
-              <h4>{t('Message Content')}</h4>
+              <h4>{t('Message content')}</h4>
             </StyledSectionTitle>
             <div className="inline-container add-margin">
               <Radio.Group onChange={onContentTypeChange} value={contentType}>
@@ -1328,7 +1328,7 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
               onChange={onDashboardChange}
             />
             <StyledSectionTitle>
-              <h4>{t('Notification Method')}</h4>
+              <h4>{t('Notification method')}</h4>
             </StyledSectionTitle>
             <NotificationMethod
               setting={notificationSettings[0]}

--- a/superset-frontend/src/views/CRUD/alert/ExecutionLog.tsx
+++ b/superset-frontend/src/views/CRUD/alert/ExecutionLog.tsx
@@ -105,7 +105,7 @@ function ExecutionLog({ addDangerToast, isReportEnabled }: ExecutionLogProps) {
             original: { start_dttm: startDttm },
           },
         }: any) => moment(new Date(startDttm)).format('MM/DD/YYYY hh:mm:ss a'),
-        Header: t('Start At'),
+        Header: t('Start at'),
         accessor: 'start_dttm',
       },
       {
@@ -124,7 +124,7 @@ function ExecutionLog({ addDangerToast, isReportEnabled }: ExecutionLogProps) {
       },
       {
         accessor: 'error_message',
-        Header: t('Error Message'),
+        Header: t('Error message'),
       },
     ],
     [isReportEnabled],

--- a/superset-frontend/src/views/CRUD/alert/components/AlertReportCronScheduler.tsx
+++ b/superset-frontend/src/views/CRUD/alert/components/AlertReportCronScheduler.tsx
@@ -80,7 +80,7 @@ export const AlertReportCronScheduler: FunctionComponent<AlertReportCronSchedule
                 name="crontab"
                 ref={inputRef}
                 style={error ? { borderColor: theme.colors.error.base } : {}}
-                placeholder={t('CRON Expression')}
+                placeholder={t('CRON expression')}
                 disabled={scheduleFormat !== 'input'}
                 onBlur={event => {
                   onChange(event.target.value);

--- a/superset-frontend/src/views/CRUD/alert/components/AlertStatusIcon.tsx
+++ b/superset-frontend/src/views/CRUD/alert/components/AlertStatusIcon.tsx
@@ -59,27 +59,27 @@ export default function AlertStatusIcon({
     case AlertState.success:
       lastStateConfig.name = isReportEnabled ? 'check' : 'alert-solid-small';
       lastStateConfig.label = isReportEnabled
-        ? t('Report Sent')
-        : t('Alert Triggered, Notification Sent');
+        ? t('Report sent')
+        : t('Alert triggered, notification sent');
       lastStateConfig.status = AlertState.success;
       break;
     case AlertState.working:
       lastStateConfig.name = 'running';
       lastStateConfig.label = isReportEnabled
-        ? t('Report Sending')
-        : t('Alert Running');
+        ? t('Report sending')
+        : t('Alert running');
       lastStateConfig.status = AlertState.working;
       break;
     case AlertState.error:
       lastStateConfig.name = 'x-small';
       lastStateConfig.label = isReportEnabled
-        ? t('Report Failed')
-        : t('Alert Failed');
+        ? t('Report failed')
+        : t('Alert failed');
       lastStateConfig.status = AlertState.error;
       break;
     case AlertState.noop:
       lastStateConfig.name = 'check';
-      lastStateConfig.label = t('Nothing Triggered');
+      lastStateConfig.label = t('Nothing triggered');
       lastStateConfig.status = AlertState.noop;
       break;
     case AlertState.grace:
@@ -89,7 +89,7 @@ export default function AlertStatusIcon({
       break;
     default:
       lastStateConfig.name = 'check';
-      lastStateConfig.label = t('Nothing Triggered');
+      lastStateConfig.label = t('Nothing triggered');
       lastStateConfig.status = AlertState.noop;
   }
   return (

--- a/superset-frontend/src/views/CRUD/annotation/AnnotationList.tsx
+++ b/superset-frontend/src/views/CRUD/annotation/AnnotationList.tsx
@@ -177,14 +177,14 @@ function AnnotationList({
           const actions = [
             {
               label: 'edit-action',
-              tooltip: t('Edit Annotation'),
+              tooltip: t('Edit annotation'),
               placement: 'bottom',
               icon: 'edit' as IconName,
               onClick: handleEdit,
             },
             {
               label: 'delete-action',
-              tooltip: t('Delete Annotation'),
+              tooltip: t('Delete annotation'),
               placement: 'bottom',
               icon: 'trash' as IconName,
               onClick: handleDelete,
@@ -215,7 +215,7 @@ function AnnotationList({
   });
 
   subMenuButtons.push({
-    name: t('Bulk Select'),
+    name: t('Bulk select'),
     onClick: toggleBulkSelect,
     buttonStyle: 'secondary',
     'data-test': 'annotation-bulk-select',

--- a/superset-frontend/src/views/CRUD/annotation/AnnotationModal.tsx
+++ b/superset-frontend/src/views/CRUD/annotation/AnnotationModal.tsx
@@ -262,16 +262,16 @@ const AnnotationModal: FunctionComponent<AnnotationModalProps> = ({
           ) : (
             <StyledIcon name="plus-large" />
           )}
-          {isEditMode ? t('Edit Annotation') : t('Add Annotation')}
+          {isEditMode ? t('Edit annotation') : t('Add annotation')}
         </h4>
       }
     >
       <StyledAnnotationTitle>
-        <h4>{t('Basic Information')}</h4>
+        <h4>{t('Basic information')}</h4>
       </StyledAnnotationTitle>
       <AnnotationContainer>
         <div className="control-label">
-          {t('annotation name')}
+          {t('Annotation name')}
           <span className="required">*</span>
         </div>
         <input
@@ -304,7 +304,7 @@ const AnnotationModal: FunctionComponent<AnnotationModalProps> = ({
         />
       </AnnotationContainer>
       <StyledAnnotationTitle>
-        <h4>{t('Additional Information')}</h4>
+        <h4>{t('Additional information')}</h4>
       </StyledAnnotationTitle>
       <AnnotationContainer>
         <div className="control-label">{t('description')}</div>
@@ -316,7 +316,7 @@ const AnnotationModal: FunctionComponent<AnnotationModalProps> = ({
         />
       </AnnotationContainer>
       <AnnotationContainer>
-        <div className="control-label">{t('json metadata')}</div>
+        <div className="control-label">{t('JSON metadata')}</div>
         <StyledJsonEditor
           onChange={onJsonChange}
           value={

--- a/superset-frontend/src/views/CRUD/annotationlayers/AnnotationLayerModal.tsx
+++ b/superset-frontend/src/views/CRUD/annotationlayers/AnnotationLayerModal.tsx
@@ -215,17 +215,17 @@ const AnnotationLayerModal: FunctionComponent<AnnotationLayerModalProps> = ({
             <StyledIcon name="plus-large" />
           )}
           {isEditMode
-            ? t('Edit Annotation Layer Properties')
-            : t('Add Annotation Layer')}
+            ? t('Edit annotation layer properties')
+            : t('Add annotation layer')}
         </h4>
       }
     >
       <StyledAnnotationLayerTitle>
-        <h4>{t('Basic Information')}</h4>
+        <h4>{t('Basic information')}</h4>
       </StyledAnnotationLayerTitle>
       <LayerContainer>
         <div className="control-label">
-          {t('annotation layer name')}
+          {t('Annotation layer name')}
           <span className="required">*</span>
         </div>
         <input

--- a/superset-frontend/src/views/CRUD/annotationlayers/AnnotationLayersList.tsx
+++ b/superset-frontend/src/views/CRUD/annotationlayers/AnnotationLayersList.tsx
@@ -64,7 +64,7 @@ function AnnotationLayersList({
     toggleBulkSelect,
   } = useListViewResource<AnnotationLayerObject>(
     'annotation_layer',
-    t('annotation layers'),
+    t('Annotation layers'),
     addDangerToast,
   );
 
@@ -178,7 +178,7 @@ function AnnotationLayersList({
 
           return moment(utc).format(MOMENT_FORMAT);
         },
-        Header: t('Last Modified'),
+        Header: t('Last modified'),
         accessor: 'changed_on',
         size: 'xl',
       },
@@ -203,14 +203,14 @@ function AnnotationLayersList({
 
           return moment(utc).format(MOMENT_FORMAT);
         },
-        Header: t('Created On'),
+        Header: t('Created on'),
         accessor: 'created_on',
         size: 'xl',
       },
       {
         accessor: 'created_by',
         disableSortBy: true,
-        Header: t('Created By'),
+        Header: t('Created by'),
         Cell: ({
           row: {
             original: { created_by: createdBy },
@@ -263,7 +263,7 @@ function AnnotationLayersList({
     subMenuButtons.push({
       name: (
         <>
-          <i className="fa fa-plus" /> {t('Annotation Layer')}
+          <i className="fa fa-plus" /> {t('Annotation layer')}
         </>
       ),
       buttonStyle: 'primary',
@@ -275,7 +275,7 @@ function AnnotationLayersList({
 
   if (canDelete) {
     subMenuButtons.push({
-      name: t('Bulk Select'),
+      name: t('Bulk select'),
       onClick: toggleBulkSelect,
       buttonStyle: 'secondary',
     });
@@ -284,7 +284,7 @@ function AnnotationLayersList({
   const filters: Filters = useMemo(
     () => [
       {
-        Header: t('Created By'),
+        Header: t('Created by'),
         id: 'created_by',
         input: 'select',
         operator: 'rel_o_m',
@@ -320,7 +320,7 @@ function AnnotationLayersList({
       }}
     >
       <>
-        <i className="fa fa-plus" /> {t('Annotation Layer')}
+        <i className="fa fa-plus" /> {t('Annotation layer')}
       </>
     </Button>
   );
@@ -336,7 +336,7 @@ function AnnotationLayersList({
 
   return (
     <>
-      <SubMenu name={t('Annotation Layers')} buttons={subMenuButtons} />
+      <SubMenu name={t('Annotation layers')} buttons={subMenuButtons} />
       <AnnotationLayerModal
         addDangerToast={addDangerToast}
         layer={currentAnnotationLayer}

--- a/superset-frontend/src/views/CRUD/chart/ChartCard.tsx
+++ b/superset-frontend/src/views/CRUD/chart/ChartCard.tsx
@@ -69,7 +69,7 @@ export default function ChartCard({
       {canDelete && (
         <Menu.Item>
           <ConfirmStatusChange
-            title={t('Please Confirm')}
+            title={t('Please confirm')}
             description={
               <>
                 {t('Are you sure you want to delete')} <b>{chart.slice_name}</b>

--- a/superset-frontend/src/views/CRUD/chart/ChartList.tsx
+++ b/superset-frontend/src/views/CRUD/chart/ChartList.tsx
@@ -217,7 +217,7 @@ function ChartList(props: ChartListProps) {
             original: { viz_type: vizType },
           },
         }: any) => registry.get(vizType)?.name || vizType,
-        Header: t('Visualization Type'),
+        Header: t('Visualization type'),
         accessor: 'viz_type',
         size: 'xxl',
       },
@@ -244,7 +244,7 @@ function ChartList(props: ChartListProps) {
             },
           },
         }: any) => <a href={changedByUrl}>{changedByName}</a>,
-        Header: t('Modified By'),
+        Header: t('Modified by'),
         accessor: 'changed_by.first_name',
         size: 'xl',
       },
@@ -254,7 +254,7 @@ function ChartList(props: ChartListProps) {
             original: { changed_on_delta_humanized: changedOn },
           },
         }: any) => <span className="no-wrap">{changedOn}</span>,
-        Header: t('Last Modified'),
+        Header: t('Last modified'),
         accessor: 'changed_on_delta_humanized',
         size: 'xl',
       },
@@ -270,7 +270,7 @@ function ChartList(props: ChartListProps) {
           },
         }: any) =>
           createdBy ? `${createdBy.first_name} ${createdBy.last_name}` : '',
-        Header: t('Created By'),
+        Header: t('Created by'),
         accessor: 'created_by',
         disableSortBy: true,
         size: 'xl',
@@ -294,7 +294,7 @@ function ChartList(props: ChartListProps) {
             <span className="actions">
               {canDelete && (
                 <ConfirmStatusChange
-                  title={t('Please Confirm')}
+                  title={t('Please confirm')}
                   description={
                     <>
                       {t('Are you sure you want to delete')}{' '}
@@ -388,7 +388,7 @@ function ChartList(props: ChartListProps) {
       paginate: true,
     },
     {
-      Header: t('Created By'),
+      Header: t('Created by'),
       id: 'created_by',
       input: 'select',
       operator: FilterOperators.relationOneMany,
@@ -409,7 +409,7 @@ function ChartList(props: ChartListProps) {
       paginate: true,
     },
     {
-      Header: t('Viz Type'),
+      Header: t('Viz type'),
       id: 'viz_type',
       input: 'select',
       operator: FilterOperators.equals,
@@ -480,13 +480,13 @@ function ChartList(props: ChartListProps) {
     {
       desc: true,
       id: 'changed_on_delta_humanized',
-      label: 'Recently Modified',
+      label: 'Recently modified',
       value: 'recently_modified',
     },
     {
       desc: false,
       id: 'changed_on_delta_humanized',
-      label: 'Least Recently Modified',
+      label: 'Least recently modified',
       value: 'least_recently_modified',
     },
   ];
@@ -510,7 +510,7 @@ function ChartList(props: ChartListProps) {
   const subMenuButtons: SubMenuProps['buttons'] = [];
   if (canDelete || canExport) {
     subMenuButtons.push({
-      name: t('Bulk Select'),
+      name: t('Bulk select'),
       buttonStyle: 'secondary',
       onClick: toggleBulkSelect,
     });

--- a/superset-frontend/src/views/CRUD/csstemplates/CssTemplateModal.tsx
+++ b/superset-frontend/src/views/CRUD/csstemplates/CssTemplateModal.tsx
@@ -219,17 +219,17 @@ const CssTemplateModal: FunctionComponent<CssTemplateModalProps> = ({
             <StyledIcon name="plus-large" />
           )}
           {isEditMode
-            ? t('Edit CSS Template Properties')
-            : t('Add CSS Template')}
+            ? t('Edit CSS template properties')
+            : t('Add CSS template')}
         </h4>
       }
     >
       <StyledCssTemplateTitle>
-        <h4>{t('Basic Information')}</h4>
+        <h4>{t('Basic information')}</h4>
       </StyledCssTemplateTitle>
       <TemplateContainer>
         <div className="control-label">
-          {t('css template name')}
+          {t('CSS template name')}
           <span className="required">*</span>
         </div>
         <input

--- a/superset-frontend/src/views/CRUD/csstemplates/CssTemplatesList.tsx
+++ b/superset-frontend/src/views/CRUD/csstemplates/CssTemplatesList.tsx
@@ -63,7 +63,7 @@ function CssTemplatesList({
     toggleBulkSelect,
   } = useListViewResource<TemplateObject>(
     'css_template',
-    t('css templates'),
+    t('CSS templates'),
     addDangerToast,
   );
   const [cssTemplateModalOpen, setCssTemplateModalOpen] = useState<boolean>(
@@ -155,7 +155,7 @@ function CssTemplatesList({
             </TooltipWrapper>
           );
         },
-        Header: t('Last Modified'),
+        Header: t('Last modified'),
         accessor: 'changed_on_delta_humanized',
         size: 'xl',
         disableSortBy: true,
@@ -181,7 +181,7 @@ function CssTemplatesList({
 
           return moment(utc).fromNow();
         },
-        Header: t('Created On'),
+        Header: t('Created on'),
         accessor: 'created_on',
         size: 'xl',
         disableSortBy: true,
@@ -189,7 +189,7 @@ function CssTemplatesList({
       {
         accessor: 'created_by',
         disableSortBy: true,
-        Header: t('Created By'),
+        Header: t('Created by'),
         Cell: ({
           row: {
             original: { created_by: createdBy },
@@ -237,7 +237,7 @@ function CssTemplatesList({
   );
 
   const menuData: SubMenuProps = {
-    name: t('CSS Templates'),
+    name: t('CSS templates'),
   };
 
   const subMenuButtons: SubMenuProps['buttons'] = [];
@@ -246,7 +246,7 @@ function CssTemplatesList({
     subMenuButtons.push({
       name: (
         <>
-          <i className="fa fa-plus" /> {t('Css Template')}
+          <i className="fa fa-plus" /> {t('CSS template')}
         </>
       ),
       buttonStyle: 'primary',
@@ -259,7 +259,7 @@ function CssTemplatesList({
 
   if (canDelete) {
     subMenuButtons.push({
-      name: t('Bulk Select'),
+      name: t('Bulk select'),
       onClick: toggleBulkSelect,
       buttonStyle: 'secondary',
     });
@@ -270,7 +270,7 @@ function CssTemplatesList({
   const filters: Filters = useMemo(
     () => [
       {
-        Header: t('Created By'),
+        Header: t('Created by'),
         id: 'created_by',
         input: 'select',
         operator: 'rel_o_m',

